packages/conf-radare2/conf-radare2.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+name: "conf-radare2"
+version: "0.1"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bap/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bap/issues"
+dev-repo: "git://github.com/BinaryAnalysisPlatform/bap/"
+license: "MIT"
+build: [
+  ["ocaml" "unix.cma" "find-radare2.ml"]
+]
+depends: [
+   "ocaml"
+   "base-unix"
+   "conf-which" {build}
+]
+substs: [ "find-radare2.ml" ]
+flags: [ conf ]
+
+depexts: [ "radare2" ]
```

Comment:
is the name of the package `radare2` on **every single** distributions?
Could it be replaced by:
```
depexts: [
  ["radare2"] {os-family = "debian"}
  ["radare2"] {os-distribution = "fedora"}
  ["radare2"] {os-distribution = "arch"}
  ["radare2"] {os-distribution = "gentoo"}
  ["radare2"] {os-distribution = "homebrew" & os = "macos"}
]
```
(taken from the `radare2` package) instead?

Reply:
done!